rpc/src/client.rs

```diff
@@ -27,6 +29,13 @@ mod native {
 
     use crate::Error;
 
+    const MAX_SHARES_SIZE_IN_EDS: usize =
```

Comment:
this feels a bit weird, how about sth like `MAX_EDS_SIZE_BYTES`

Reply:
sure